src/item_factory.cpp

```diff
@@ -1744,11 +1757,76 @@ void Item_factory::load_pet_armor( const JsonObject &jo, const std::string &src
 
 void islot_armor::load( const JsonObject &jo )
 {
-    optional( jo, was_loaded, "encumbrance", encumber, 0 );
-    // Default max_encumbrance will be set to a reasonable value in
-    // finalize_post
-    optional( jo, was_loaded, "max_encumbrance", max_encumber, -1 );
-    optional( jo, was_loaded, "coverage", coverage, 0 );
+    if( jo.has_array( "coverage_data" ) ) {
+
+        for( const JsonObject &obj : jo.get_array( "coverage_data" ) ) {
+            random_armor_data tempData;
+            body_part_set temp_cover_data;
+            assign_coverage_from_json( obj, "parts", temp_cover_data, sided );
+            tempData.covers = temp_cover_data;
+
+            if( obj.has_array( "encumbrance" ) ) {
+                tempData.encumber = obj.get_array( "encumbrance" ).get_int( 0 );
+                tempData.max_encumber = obj.get_array( "encumbrance" ).get_int( 1 );
+            } else if( obj.has_int( "encumbrance" ) ) {
+                tempData.encumber = obj.get_int( "encumbrance" );
+                tempData.max_encumber = obj.get_int( "encumbrance" );
```

Comment:
Is this an intentional change? `max_encumber` didn't previously default to the same as `encumber` when not specified.

Reply:
It is set to either encumbrance or encumbrance + volume / 250ml in finalise_post, so it doesnt really make a difference what its value is here. I will set it to 0 though here, as that correctly mimics the previous behaviour.

Reply:
It's only set to encumbrance or encumbrance + volume / 250ml if it's negative.  The default should be -1, I thought, from when I wrote this code originally, but I can't find that right now...